lib/kadai-core/src/main/java/io/kadai/spi/routing/api/TaskRoutingProvider.java

```diff
@@ -65,5 +69,38 @@ public interface TaskRoutingProvider {
    * @param task the {@linkplain Task} for which a {@linkplain Workbasket} must be determined
    * @return the {@linkplain Workbasket#getId() id} of the {@linkplain Workbasket}
    */
-  String determineWorkbasketId(Task task);
+  default String determineWorkbasketId(Task task) {
+    return null;
+  }
+
+  /**
+   * This method is used instead of {@linkplain TaskRoutingProvider#determineWorkbasketId(Task)} if
```

Comment:
Shoudl we add a little bit more details here? Like "Use this if you want to initialize the owner during Routing"

And didn't you have a test that the owner is not overwritten if existing? Maybe we should specify this behaviour as well?

Reply:
I will add this information :)